### ISSUE TITLE
generative-art-opensource

### ISSUE THREAD

**Issue by R777R** [HashLips/generative-art-opensource#71]
Is there anyway to add automatically generated numeric and alphabetic characters or unique serial number in all 10,000 output .PNG's?

Thank you

**Comment by Ponda1** [comment]
Yeah there is a signImage in the code which can be used to sign the image with a number(default) or you can edit it to sign something else

**Comment by R777R** [comment]
> Yeah there is a signImage in the code which can be used to sign the image with a number(default) or you can edit it to sign something else

Hello, I saw some of your other helpful comments for art generative code, can you please tell me What is really different between generative art node & generative art open source, which one is easy to generate and upload the nfts to the platforms. Thank you 🙏
